src/test/java/uk/gov/pay/api/it/rule/RedisContainer.java

```diff
@@ -0,0 +1,118 @@
+package uk.gov.pay.api.it.rule;
+
+import com.google.common.base.Stopwatch;
+import com.spotify.docker.client.DockerClient;
+import com.spotify.docker.client.LogStream;
+import com.spotify.docker.client.exceptions.DockerException;
+import com.spotify.docker.client.messages.ContainerConfig;
+import com.spotify.docker.client.messages.ContainerInfo;
+import com.spotify.docker.client.messages.HostConfig;
+import com.spotify.docker.client.messages.LogConfig;
+import com.spotify.docker.client.messages.PortBinding;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import redis.clients.jedis.Jedis;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static java.lang.Integer.parseInt;
+import static java.util.stream.Collectors.joining;
+
+public class RedisContainer {
+
+    private static final Logger logger=LoggerFactory.getLogger(RedisContainer.class);
+
+    private final String containerId;
+    private final int port;
+    private DockerClient docker;
+    private String host;
+    private volatile boolean stopped=false;
+
+    private static final int DB_TIMEOUT_SEC=10;
+    private static final String REDIS_IMAGE="redis:latest";
+    private static final String INTERNAL_PORT="6379";
+
+    public RedisContainer(DockerClient docker, String host) throws DockerException, InterruptedException, IOException, ClassNotFoundException {
+
+        this.docker=docker;
+        this.host=host;
```

Comment:
spaces. `this.host = host` and similar for docker line above

Reply:
Done..